_docs/_latest/configure/notifier/index.md

```diff
@@ -88,6 +88,22 @@ accordingly.
       data_format: csv
   ```
 
+#### YAML below shows the email connector config for Mailjet.
+
+To configure other email connector, `name` and `auth` fields should be modified
+accordingly.
+
+  ```yaml
+  notifier:
+    email_connector:
+      name: sendgrid
+      auth:
+        api_key: {Mailjet_API_KEY}
+        mailjet_api_key: {Mailjet_API_KEY}
```

Comment:
What is the difference between `api_key` and `mailjet_api_key`?  Are both needed?  Or just one of them?

Reply:
just one of them api_key is ok